.github/workflows/publish-sdist.yml

```diff
@@ -10,9 +10,9 @@ jobs:
     name: Build and publish on PyPI
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@master
+      - uses: actions/checkout@v3
       - name: Set up Python 3.8
-        uses: actions/setup-python@v2
+        uses: actions/setup-python@v4
         with:
           python-version: 3.8
 
```

Comment:
We could set a version also for `pypa/gh-action-pypi-publish@master`

Reply:
thank you; great eye.